_journal-configs/nature-reviews-drug-discovery.yaml

```
@@ -0,0 +1,28 @@
+name: Nature Reviews Drug Discovery
+publisher: Springer Nature
+url: https://www.nature.com/nrd/about/figure-guide
+
+sizes:
+  single_column: { width_mm: 88,  max_height_mm: 240 }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)